packages/query/src/block-processor.ts

```diff
@@ -162,12 +162,12 @@ export class BlockProcessor implements BlockProcessorInterface {
 
     // special case genesis sync
```

Comment:
issue: Can we add more to this code comment? Something like:

```
// Special case if no syncing has yet been done. Will prep for syncing 
// and see if there's a bundled genesis block passed (saves on network request time)
```

Reply:
changed